<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string>Sivut</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>445</width>
         <height>686</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Nimi</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="nameLineEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Kuvaus</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTextEdit" name="descTextEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>150</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Käyttö</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPlainTextEdit" name="commandTextEdit"/>
           </item>
           <item>
            <widget class="QTextEdit" name="parametreTextEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Esimerkki</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTextEdit" name="exampleTextEdit">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTiedosto">
    <property name="title">
     <string>Tiedosto</string>
    </property>
    <addaction name="actionUusi"/>
    <addaction name="actionAvaa"/>
    <addaction name="separator"/>
    <addaction name="actionTallenna"/>
    <addaction name="actionTallenna_nimell"/>
    <addaction name="separator"/>
    <addaction name="actionPoistu"/>
   </widget>
   <widget class="QMenu" name="menuLuo">
    <property name="title">
     <string>Luo</string>
    </property>
    <addaction name="actionHtml_sivut"/>
   </widget>
   <addaction name="menuTiedosto"/>
   <addaction name="menuLuo"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUusi">
   <property name="text">
    <string>Uusi...</string>
   </property>
  </action>
  <action name="actionTallenna">
   <property name="text">
    <string>Tallenna</string>
   </property>
  </action>
  <action name="actionAvaa">
   <property name="text">
    <string>Avaa...</string>
   </property>
  </action>
  <action name="actionTallenna_nimell">
   <property name="text">
    <string>Tallenna nimellä...</string>
   </property>
  </action>
  <action name="actionPoistu">
   <property name="text">
    <string>Poistu</string>
   </property>
  </action>
  <action name="actionHtml_sivut">
   <property name="text">
    <string>Html - sivut...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
